### PR TITLE
OCPBUGS-43481: feat: add unreachable kubeapi intervals for duplicate events exceptions

### PR DESCRIPTION
**Pull request by eggfoobar**

### CODE REVIEW

File: pkg/monitortestlibrary/pathologicaleventlibrary/duplicated_event_patterns.go [hunk 1]
```diff
@@ -992,14 +992,24 @@ func newSingleNodeKubeAPIProgressingEventMatcher(finalIntervals monitorapi.Inter
...
+		// KubeAPI is progressing
+		case eventInterval.Message.Reason == monitorapi.NodeInstallerReason &&
+			eventInterval.Source == monitorapi.SourceOperatorState &&
+			eventInterval.Locator.Keys[monitorapi.LocatorClusterOperatorKey] == "kube-apiserver" &&
+			eventInterval.Message.Annotations[monitorapi.AnnotationCondition] == "Progressing":
+			return true
 
-		isKubeAPIInstaller := isNodeInstaller && isOperatorSource && isKubeAPI
-		isKubeAPIInstallProgressing := isKubeAPIInstaller && eventInterval.Message.Annotations[monitorapi.AnnotationCondition] == "Progressing"
+		// KubeAPI unreachable
```
Comment (deads2k) [inline pkg/monitortestlibrary/pathologicaleventlibrary/duplicated_event_patterns.go:1003]: no

**Comment by deads2k** [comment]
/hold

Let's figure out why the health chekc of a controller is timing out instead.

**Comment by eggfoobar** [comment]
/close 
in favor of https://github.com/openshift/machine-api-operator/pull/1299

**Pull request closed without merging**